package.json

```diff
@@ -32,12 +32,13 @@
     "@babel/preset-react": "^7.9.4",
     "@babel/standalone": "^7.9.5",
     "@emotion/core": "^10.0.27",
-    "@guardian/src-button": "^0.18.1",
-    "@guardian/src-ed-lines": "^0.18.1",
-    "@guardian/src-foundations": "^0.18.1",
+    "@guardian/src-brand": "^1.4.0",
+    "@guardian/src-button": "^1.4.0",
+    "@guardian/src-ed-lines": "^1.4.0",
+    "@guardian/src-foundations": "^1.4.0",
     "@guardian/src-icons": "^1.4.0",
-    "@guardian/src-svgs": "^0.18.1",
-    "@guardian/src-text-input": "^0.18.1",
+    "@guardian/src-svgs": "^1.4.0",
```

Comment:
Do we need src-svgs as well as src-icons? I believe [they are identical](https://github.com/guardian/source/pull/367) (but src-svgs is deprecated).

Reply:
Yes good point Rupert - I will remove this 👍

Reply:
Though actually on checking this, these are used in the contributions banners and buttons, and it seems a bit cheeky for me to go in an update them without consulting anyone! So Maybe I'll just let them know instead.